### PR TITLE
[frontend] Add Note resolution in TimeLine (#3486)

### LINKED ISSUE #3486: Resolve Note's name in the Timeline view of a Report & Cases
## Use case

Note can be used to add unstructured context in a Report or a Case. But Note's name are not displayed in the Timeline View.
image

Being able to add text (through Note) in those view participate to the capacity to automaticaly generate outcomes from OpenCTI Knowledge.

That's because the Note contained the Report/Case in STIX model. Thus, the name of the note is not resolved.

## Current Workaround

None

## Proposed Solution

create a resolver for the Note's name

### PR DESCRIPTION
**Pull request by RomuDeuxfois**
### Proposed changes

* Add missing inline fragments for Note in different TimeLine

### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/3486

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by RomuDeuxfois** [comment]
Reflexion : @Kedae could we have a reusable fragment containing all the definitions of our entities ? To mutalize the fragment present in the timelines for example.


---
@Kedae : Yes we could, we probably should and may be we will, but that's a reflexion we will have globally with Julien, some similar works has been done on report I think. We will think about that in the future.

File: opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ const LassoSelection: FunctionComponent<LassoSelectionProps> = ({
     coord = reposition(e);
     currentContext.moveTo(coord.x, coord.y);
     currentContext.lineWidth = 1;
-    currentContext.setLineDash([0.5, 1]);
+    currentContext.setLineDash([1, 3]);
```
Comment (SarahBocognano) [inline opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx:82]: Why did you change that line ? (curious)

**Review by SarahBocognano — APPROVED** [review]
Tested ✅

File: opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx (hunk 1 above)
Comment (RomuDeuxfois) [reply opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx:82]: @Kedae is the real author of this

Comment (Kedae) [reply opencti-platform/opencti-front/src/utils/graph/LassoSelection.tsx:82]: Because I love css :) It's juste adjustment
